client/src/components/judging/JudgingRoom.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react'
+import roomData = require('../../data.json')
+
+
+interface JudgingRoomProps {
+    table: string
+}
+
+class JudgingRoom extends React.Component<JudgingRoomProps, {}>{
+    constructor(props: JudgingRoomProps) {
+        super(props)
+    }
+    public render() {
+        let index = 0
+        const tableColor = this.props.table.split(" ")[0].toLowerCase()
+        if (tableColor === "green") {
+            index = 1
+        } else if (tableColor === "orange") {
+            index = 2
+        }
+        let tableId = 0
+        let widthSize = "817.33331"
+        let heightSize = "486.6666"
+        if (window.innerWidth < 853) {
```

Comment:
Below a certain width causes issues for almost all mobile devices. The image adds a horizontal scroll element below a certain width. 

This is especially prevalent on iPhone 6,7,8,X, as below

![image](https://user-images.githubusercontent.com/5157707/55254327-f2155100-522d-11e9-9198-d93bf5cb5a92.png)
![image](https://user-images.githubusercontent.com/5157707/55254338-f7729b80-522d-11e9-9e04-066ac121c1b2.png)